app/middleware/locals.js

```diff
@@ -15,10 +15,8 @@ module.exports = config =>
     res.locals.origin = queryMapper.mapServiceChoice(req.query);
     res.locals.location = req.query.location;
     res.locals.locationHeading = queryMapper.getLocationHeading(req.query);
-    if (res.locals.location) {
-      res.locals.resultsHeading = queryMapper.getResultsHeading(req.query, res.locals.location);
-    }
-    res.locals.resultsPara = queryMapper.getResultsPara(req.query);
+    res.locals.resultsHeading = queryMapper.getResultsHeading(req.query, res.locals.location);
+    res.locals.resultsPara = queryMapper.getResultsExplanation(req.query);
```

Comment:
Can `resultsPara` be renamed inline with the improved naming of the function please?